ingestion/moj_publications.yaml

```diff
@@ -0,0 +1,26 @@
+source:
+  type: ingestion.moj_statistical_publications_source.source.MojPublicationsAPISource
+  config:
+    base_url: "https://www.gov.uk/api"
+    default_contact_email: statistics.enquiries@justice.gsi.gov.uk
+    collections_to_exclude: [
+      "offender-management-statistics-quarterly--3"
+    ]
+    access_requirements: You are free to re-use the data internally or externally without requesting permission.
+    params:
```

Comment:
I think the API params would make more sense as part of the code or API class

Reply:
i suppose we are not likely to ever want to change these?

Reply:
Could be a future where eople wan tot ingest interesting publciations from other departments- homeoffice police publiucations maybe? Let's keep it here, they are parameters after all

Reply:
have moved some of these to the code